### PR TITLE
[#1873298] implement azure-plugin-storage-ide-lib which would be shared between eclipse and intellij

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
[AB#1873298](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1873298)

### CODE REVIEW

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class StorageActionsContributor implements IActionsContributor {
+    public static final String SERVICE_ACTIONS = "actions.storage.service";
+    public static final String ACCOUNT_ACTIONS = "actions.storage.account";
+
+    public static final Action.Id<StorageAccount> OPEN_STORAGE_BROWSER = Action.Id.of("action.storage.open_storage_explorer");
+
+    @Override
+    public void registerActions(AzureActionManager am) {
+        final Consumer<StorageAccount> open = OpenStorageBrowserAction::openStorageBrowser;
+        final ActionView.Builder openView = new ActionView.Builder("Open Storage Explorer", "/icons/action/portal.svg")
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java:31]: Storage Explorer -> Open In Portal

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/OpenStorageBrowserAction.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.microsoft.azure.toolkit.lib.common.model.Subscription;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azure.toolkit.lib.storage.service.StorageAccount;
+
+public class OpenStorageBrowserAction {
+    public static void openStorageBrowser(StorageAccount account) {
+        final AzureString title = AzureString.format("storage|account.open_storage_explorer", account.name());
+        AzureTaskManager.getInstance().runInBackground(new AzureTask<>(title, () -> {
+            final Subscription subscription = Azure.az(AzureAccount.class).account().getSubscription(account.subscriptionId());
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/OpenStorageBrowserAction.java]: May remove this line

Comment (wangmingliang-ms) [reply Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/OpenStorageBrowserAction.java]: fixed

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java (hunk 1 above)
Comment (wangmingliang-ms) [reply Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java:31]: fixed.

**Review by Flanker32 — APPROVED** [review]

**Pull request merged**